compiler_opt/tools/extract_ir_lib.py

```diff
@@ -135,9 +135,13 @@ def _extract_clang_artifacts(self, llvm_objcopy_path: str, cmd_filter: str,
       return None
     os.makedirs(self.dest_dir(), exist_ok=True)
     try:
+      subprocess_output = None if logging.get_verbosity(
```

Comment:
why?

Reply:
If there is a failure, `llvm-objcopy` will log it to `STDERR`/`STDOUT` and that will go to the standard output/error of the python process if we don't specify this so the user will end up seeing all of the error messages from `llvm-objcopy` despite specifying `--verbosity=-1` or `--verbosity=-2`.

Reply:
wouldn't a better handling be capturing `llvm-objcopy`'s output and `logging.<whatever>` it?

Reply:
Yep, we can do that too. That would probably be better because it gives us explicit control over the level that we log it at.

Reply:
Should be updated now. I had to refactor to check the return code of the subprocess and fail on that rather than catch an exception in a try except block, but it preserves the functionality and isn't too big of a change.